tuning/scripts/analyze_results.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
```

Comment:
Just from an outside perspective, why is this a bash script and not a python script? Argument parsing, file enumeration and calls to other scripts would be drastically simplified. Not to mention portability would now be extended to any platform running Python and would be more consistent.